com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfRobotExtensions.cs

```diff
@@ -0,0 +1,363 @@
+﻿/*
+© Siemens AG, 2018
+Author: Suzannah Smith (suzannah.smith@siemens.com)
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+<http://www.apache.org/licenses/LICENSE-2.0>.
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System.Collections.Generic;
+using System;
+using System.IO;
+using System.Linq;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+using UnityEngine;
+
+namespace RosSharp.Urdf
+{
+    public static class UrdfRobotExtensions
+    {
+        static string tagName = "robot";
+        static string collisionObjectName = "Collisions";
+        public static ImportSettings importsettings;
+
+        public static void Create()
+        {
+            CreateTag();
+            GameObject robotGameObject = new GameObject("Robot");
+
+            robotGameObject.tag = tagName;
+            robotGameObject.AddComponent<UrdfRobot>();
+            robotGameObject.AddComponent<RosSharp.Control.Controller>();
+
+            UrdfPlugins.Create(robotGameObject.transform);
+
+            UrdfLink urdfLink = UrdfLinkExtensions.Create(robotGameObject.transform).GetComponent<UrdfLink>();
+            urdfLink.name = "base_link";
```

Comment:
I may misunderstand it. I am curious why we want to always the name of the link to `base_link` instead of loading the name of the link from the URDF file?

Reply:
I believe this is  an initial default name in case the link had no name. If the link has a name, it will later be changed to that:

Reply:
![Screen Shot 2021-04-01 at 10 01 34 AM](https://user-images.githubusercontent.com/11860247/113328927-c2dcc300-92d1-11eb-9240-7cd886272ad6.png)